xbmc/interfaces/builtins/PlayerBuiltins.cpp

```diff
@@ -518,7 +518,7 @@ int PlayOrQueueMedia(const std::vector<std::string>& params, bool forcePlay)
     item.SetProperty("check_resume", false);
   }
 
-  if (item.m_bIsFolder || item.IsPlayList())
+  if (item.m_bIsFolder || item.IsPlayList() || !forcePlay)
```

Comment:
Maybe place the forceplay check before the isplaylist check? I mean, the playlist check should be a bit more intensive so in the cases force play is not set it should save a few CPU cycles.

Reply:
I need to rethink the naming of the variable `forcePlay` - actually it should be named 'queueItem' and checks adapted to the inverse semantics.